src/tests/eventpipe/EventPipe.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Should this one run against 7?

Reply:
Should it run on all 3? netcoreapp3.1, net6.0 and net7.0? Should I add net7.0 to UnitTestTargetFrameworks in Directory.Build.props and use that property?

Reply:
Not all make sense to multitarget (a lot of the ones that shell out, the target is what matters. This one I think checks for things that are not shelling out but tries to test stuff that's runtime dependent? Would need to check closer if this is the suite I am thinking of.

Reply:
As it turns out, running the EventPipe.UnitTests on net7.0 found a problem or at least it fails on net7.0 (but passes on the rest of the target frameworks).